1_fetch/src/get_feature_vars.R

```diff
@@ -198,9 +268,9 @@ prep_feature_vars <- function(sb_var_data, sites_all, sites_screened,
                                  class == 7 ~ "BARREN", class == 8 ~ "FOREST", 
                                  class == 9 ~ "FOREST", class == 10 ~ "FOREST", 
                                  class == 11 ~ "GRASSLAND", class == 12 ~ "SHRUBLAND", 
-                                 class == 13 ~ "CROPLAND", class == 14 ~ "HAY/PASTURE", 
+                                 class == 13 ~ "CROPLAND", class == 14 ~ "HAYPASTURE", 
```

Comment:
In [this issue](https://github.com/USGS-R/regional-hydrologic-forcings-ml/pull/163/files) we convert these from `/` to `-`.
```suggestion
                                 class == 13 ~ "CROPLAND", class == 14 ~ "HAY-PASTURE", 
```

Reply:
Yes but sometimes the "-" in a column header can cause issues with dplyr functions (reads it as a minus sign) so I chose to delete completely (updated for the g2 and conus feature variable data frames).